tests/integration/data/empty_proj/compile

```diff
@@ -0,0 +1 @@
+../../target/release/plc build --target=x86_64-unknown-linux-gnu  --debug  conf/plc.json
```

Comment:
what is this file used for?

Reply:
it's accidently committed, i was using a script to compile in tests :)